pkg/alerts/muting_rules.go

```diff
@@ -121,13 +121,13 @@ type MutingRuleScheduleCreateInput struct {
 
 // MutingRuleScheduleUpdateInput is the time window when the MutingRule should actively mute violations for Update
 type MutingRuleScheduleUpdateInput struct {
```

Comment:
I'm surprised this file isn't code generated, but in this case it is fortunate for now.  We'll need to solve this on the generation side as well so we can move away from the hand-crafted structs.